packages/cli/src/lib/cmds/build.ts

```diff
@@ -14,10 +14,11 @@
  *  limitations under the License.
  */
 
-import {AndroidSdkTools, Config, GradleWrapper, JdkHelper, Log, TwaManifest}
-  from '@bubblewrap/core';
+import {AndroidSdkTools, Config, DigitalAssetLinks, GradleWrapper, JdkHelper, Log, TwaManifest,
+  KeyTool} from '@bubblewrap/core';
```

Comment:
Is there an ordering to these imports? If they're alphabetical, `KeyTool` is in the wrong place.

Reply:
VS Code doesn't automatically sort it, do I've been doing this manually after auto-imports. Forgot about this one.